backend/src/controllers/__tests__/CMSLoginController.unit.test.ts

```diff
@@ -0,0 +1,109 @@
+import Server from "../../App";
+import supertest from "supertest";
+import { MongoClient, Db, Collection } from "mongodb";
+import { UNAUTHORISED_TEXT, BAD_REQUEST_TEXT } from "../../constants";
+
+/**
+ * This test utilises [supertest]{@link https://www.npmjs.com/package/supertest}
+ * and initialises the full server to perform the test per test.
+ *
+ * @author ale8k
+ */
+describe("CMSLoginController", () => {
+    let connection: MongoClient;
+    const collectionName = "cms-users";
+    let db: Db;
+    let collection: Collection;
+
+    beforeAll(async () => {
+        process.env.PORT = "3003";
+        process.env.DB_URL = global.__MONGO_URI__;
+        process.env.DB_NAME = global.__MONGO_DB_NAME__;
+        process.env.BCRYPT_SALT = "12";
+        process.env.JWT_KEY = "iamakeylol";
+        process.env.JWT_EXPIRY = "2 days";
+
+        connection = await MongoClient.connect((global.__MONGO_URI__), {
+            poolSize: 10,
+            useNewUrlParser: true,
+            useUnifiedTopology: true
+        });
+
+        db = await connection.db(global.__MONGO_DB_NAME__);
+        collection = db.collection(collectionName);
+        await collection.insert(
+            {
+                username: "alex",
+                password: "$2b$12$Cp/IwyOUsKiJZTANWklBB.k4mv07lIV1gSLT4FbtsOI.eoFi2qfTu"
+            }
```

Comment:
why are passing your crendetials here? it should be here from `.env`. This way we can't move this project  to local.